apps/docs/content/guides/getting-started/tutorials/with-expo-react-native-social-auth.mdx

```diff
@@ -0,0 +1,1386 @@
+  });
+
+  if (!loaded) {
+    // Async font loading only occurs in development.
+    return null;
+  }
+
+  return (
+    <ThemeProvider value={colorScheme === 'dark' ? DarkTheme : DefaultTheme}>
+      <AuthProvider>
+        <SplashScreenController />
+        <RootNavigator />
+        <StatusBar style="auto" />
+      </AuthProvider>
+    </ThemeProvider>
+  );
+}
+```
+
+</$CodeTabs>
+
+You can now test the app by running:
+
+```bash
+npx expo prebuild
+npx expo start --clear
+```
+
+Verify that the app works as expected. The splash screen displays while fetching the user profile, and the login page appears even when attempting to navigate to the home screen using the `Link` button.
+
+<Admonition type="note">
+
+By default Supabase Auth requires email verification before a session is created for the user. To support email verification you need to [implement deep link handling](/docs/guides/auth/native-mobile-deep-linking?platform=react-native)!
+
+While testing, you can disable email confirmation in your [project's email auth provider settings](/dashboard/project/_/auth/providers).
+
+</Admonition>
+
+## Integrate social authentication
+
+Now integrate social authentication with Supabase Auth, starting with Apple authentication.
+If you only need to implement Google authentication, you can skip to the [Google authentication](#google-authentication) section.
+
+### Apple authentication
+
+Start by adding the button inside the login screen:
+
+<$CodeTabs>
+
+```tsx name=app/login.tsx
+...
+import { AppleSignInButton } from '@/components/social-auth-buttons/apple/apple-sign-in-button';
+...
+export default function LoginScreen() {
+  return (
+    <>
+      <Stack.Screen options={{ title: 'Login' }} />
+      <ThemedView style={styles.container}>
+        ...
+        <AppleSignInButton />
+        ...
+      </ThemedView>
+    </>
+  );
+}
+...
+```
+
+</$CodeTabs>
+
+For Apple authentication, you can choose between:
+
+- [Invertase's React Native Apple Authentication library](https://github.com/invertase/react-native-apple-authentication) - that supports iOS, Android
+- [react-apple-signin-auth](https://react-apple-signin-auth.ahmedtokyo.com/) - that supports Web, also suggested by Invertase
+- [Expo's AppleAuthentication library](https://docs.expo.dev/versions/latest/sdk/apple-authentication/) - that supports iOS only
+
+For either option, you need to obtain a Service ID from the [Apple Developer Console](https://supabase.com/docs/guides/auth/social-login/auth-apple?queryGroups=framework&framework=nextjs&queryGroups=platform&platform=web#configuration-web).
+
+<Admonition type="note">
+
+To enable Apple sign-up on Android and Web, you also need to register the tunnelled URL (e.g., `https://arnrer1-anonymous-8081.exp.direct`) obtained by running:
+
+```bash
+npx expo start --tunnel
+```
+{/* TODO: HERE */}
+And add it to the "Allowed Callback URLs" list in your Supabase dashboard Apple provider configuration.
```

Comment:
@massimodeluisa Another little one… You mention the dashboard, but this field doesn't exist, and the auth team says it's only configurable via the CLI?

https://github.com/supabase/cli/blob/develop/pkg/config/templates/config.toml#L271-L272

Or did you mean something else?